docs/pages/user-docs/dashboard-page.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: default
+title: Dashboard
+parent: User Guide
+nav_order: 610
+---
+# :newspaper: Dashboard  
+
```

Comment:
Maybe you wanted to add the numbers from the picture to the corresponding table entry or sentence?

Reply:
I don't understand your comment. Could you please rephrase it?

Reply:
The screenshot of the dashboard features 6 numbers. But those numbers are not used anywhere else. It would be cool if the number would be used throughout the guide to reference the place on the screenshot to a certain description

Reply:
@PhoebeNgnfld do you still want to add this?

Reply:
I deleted the screenshot